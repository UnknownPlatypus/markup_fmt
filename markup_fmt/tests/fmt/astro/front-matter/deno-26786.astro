---
/* https://example.com */
const foo = "bar"
---
<p>Hello world</p>
