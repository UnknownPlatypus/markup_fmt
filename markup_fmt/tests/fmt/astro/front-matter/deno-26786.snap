---
source: markup_fmt/tests/fmt.rs
---
---
/* https://example.com */
const foo = "bar"
---
<p>Hello world</p>
